src/lambda_vm/ast.mli

```diff
@@ -0,0 +1,50 @@
+type ident = string
+
+type prim =
+  | Neg
+  | Add
+  | Sub
+  | Mul
+  | Div
+  | Rem
+  | And
+  | Or
+  | Xor
+  | Lsl
+  | Lsr
+  | Asr
+
+type expr =
+  (* calculus *)
+  | Var   of ident
+  | Lam   of ident * expr
```

Comment:
why not De Bruijn?

Reply:
As we discussed in private, maybe in the future we should do De Bruijn for the IR, but it makes easier for developers to reason on top of the version that contains the names.